front/lib/api/redis.ts

```diff
@@ -16,15 +33,22 @@ export async function getRedisClient(): Promise<RedisClientType> {
       url: REDIS_URI,
       isolationPoolOptions: {
         acquireTimeoutMillis: 10000, // Max time to wait for a connection: 10 seconds.
+        min: 1,
         max: 500, // Maximum number of concurrent connections for streaming.
         evictionRunIntervalMillis: 15000, // Check for idle connections every 15 seconds.
         idleTimeoutMillis: 30000, // Connections idle for more than 30 seconds will be eligible for eviction.
       },
     });
     client.on("error", (err) => logger.info({ err }, "Redis Client Error"));
     client.on("ready", () => logger.info({}, "Redis Client Ready"));
-    client.on("connect", () => logger.info({}, "Redis Client Connected"));
-    client.on("end", () => logger.info({}, "Redis Client End"));
+    client.on("connect", () => {
+      logger.info({}, "Redis Client Connected");
```

Comment:
what about adding the origin in the info log? could help zooming in on a few connections (& maybe add a few other stuff in there too)

Reply:
Added!